corehq/apps/userreports/templates/userreports/reportbuilder/configure_report.html

```diff
@@ -31,6 +32,29 @@
   {{ block.super }}
 {% endblock %}
 
+{% block additional_initial_page_data %}{{ block.super }}
+    {% initial_page_data 'column_options' column_options %}
+    {% initial_page_data 'initial_columns' initial_columns %}
+    {% initial_page_data 'application' application %}
+    {% initial_page_data 'source_id' source_id %}
+    {% initial_page_data 'source_type' source_type %}
+    {% initial_page_data 'report_preview_url' report_preview_url %}
+    {% initial_page_data 'preview_datasource_id' preview_datasource_id %}
+    {% initial_page_data 'existing_report_type' existing_report_type %}
+    {% initial_page_data 'report_title' report_title|escapejs %}
+    {% initial_page_data 'report_description' report_description|escapejs %}
```

Comment:
I don't think the `escapejs` should be necessary anymore.

Reply:
Is this because initial_page_data does it automatically or because we don't need to escape it?

Reply:
Kind of both: it no longer needs js escaping because it's no longer being plopped directly into javascript source...it does now need html attribute escaping but initial_page_data handles that.